editor/plugins/lightmap_gi_editor_plugin.cpp

```diff
@@ -166,19 +169,30 @@ void LightmapGIEditorPlugin::bake_func_end(uint64_t p_time_started) {
 }
 
 void LightmapGIEditorPlugin::_bind_methods() {
-	ClassDB::bind_method("_bake", &LightmapGIEditorPlugin::_bake);
+	ClassDB::bind_method("_bake", &LightmapGIEditorPlugin::_bake, DEFVAL(false));
 }
 
 LightmapGIEditorPlugin::LightmapGIEditorPlugin() {
+	bake_preview = memnew(Button);
+	bake_preview->set_theme_type_variation("FlatButton");
+	bake_preview->set_icon(EditorNode::get_singleton()->get_editor_theme()->get_icon(SNAME("BakePreview"), EditorStringName(EditorIcons)));
+	bake_preview->set_tooltip_text(TTR("Bakes lightmaps with low-quality settings for quick iteration.\nPreview bake quality can be changed in the Rendering > Lightmapping > Preview Bake section of the Project Settings."));
+	bake_preview->set_text(TTR("Preview Bake"));
+	bake_preview->hide();
+	bake_preview->connect("pressed", callable_mp(this, &LightmapGIEditorPlugin::_bake).bind(true));
+	add_control_to_container(CONTAINER_SPATIAL_EDITOR_MENU, bake_preview);
+
 	bake = memnew(Button);
 	bake->set_theme_type_variation("FlatButton");
 	// TODO: Rework this as a dedicated toolbar control so we can hook into theme changes and update it
 	// when the editor theme updates.
```

Comment:
This comment should probably be moved before both buttons now, since it is relevant to both of them.